packages/chain-adapters/src/cosmossdk/thorchain/types.ts

```diff
@@ -0,0 +1,13 @@
+import { HDWallet } from '@shapeshiftoss/hdwallet-core'
+import { BIP44Params } from '@shapeshiftoss/types'
+
+export type BuildDepositTxInput = {
+  memo: string
+  value: string
+  wallet: HDWallet
+  gas: string
+  fee: string
+  bip44Params: BIP44Params
+  // for synth support:
+  // asset?: AssetId // synths/ibc?
```

Comment:
remove if we're not using yet or plan to?

Reply:
07c41f9764d6207cd5323bbef93f8da0eb1d912c